tests/ui/should-panic-attribute/expected-panics/expected

```diff
@@ -0,0 +1,4 @@
+ ** 2 of 2 failed\
+Failed Checks: panicked on the `if` branch!
+Failed Checks: panicked on the `else` branch!
+VERIFICATION:- SUCCESSFUL (encountered one or more panics as expected)
```

Comment:
Missing a new line at the end of `expected` files.

Reply:
Done.